test/test.command.executableSubcommandAlias.help.js

```diff
@@ -7,7 +7,7 @@ var exec = require('child_process').exec
 var bin = path.join(__dirname, './fixtures/pm')
 
 // success case
-exec(bin + ' help', function (error, stdout, stderr) {
+exec('node ' + bin + ' help', function (error, stdout, stderr) {
```

Comment:
Another option here is to have this project use `execa` instead of child_process.exec(). This is what Jest uses. It has "smarts" to read the shebang at the top of the .js file (to find "node"). You would change this to something like this:

```
const {stdout, stderr} = execa.sync(bin, [ 'help' ]);

stdout.should.containEql('install|i');
stdout.should.containEql('search|s');
stdout.should.containEql('cache|c');
stdout.should.containEql('list');
stdout.should.containEql('publish|p');
stdout.should.not.containEql('pm|');
```

Reply:
Do you know if `execa` plays nicely with symlinks? I gave `execa` a go in `test.command.executableSubcommand.js` and it didn't like the `pmlink` test.

Reply:
Just on Windows? Or all platforms? I would have expected it to work, but that's not saying much considering the complexity of some of the stuff :)

Reply:
Only tried it locally on Windows.